test/__snapshots__/index.test.js.snap

```diff
@@ -69,6 +69,18 @@ Array [
 `;
 
 exports[`Error Messages Customized 1`] = `
+Object {
```

Comment:
This change should not alter the customized snapshots, customizing means to overwrite the `{Error}` messages via the schema (user defined). See the README and https://github.com/webpack-contrib/schema-utils/blob/master/test/fixtures/errors/schema.json for an example. If we change the default `Ajv` messages that's still the default from a user perspective :)

Reply:
This therefore also needs to be tested when using a customized `err.message` in the JSON Schema